### PR TITLE
Postgres database setup for staging and production

### PR DESCRIPTION
**Pull request by elcorbs**

### CODE REVIEW

File: terraform/development/main.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# INSTRUCTIONS:
```
Comment (louisebolden) [inline terraform/development/main.tf]: I wonder if some of the instructional comments here can be removed, assuming they now refer to a completed step?

Some of these comments are perhaps more informational/relevant to future developers opening this file (e.g. line 7?) so may be important to keep.

File: terraform/production/main.tf [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# INSTRUCTIONS:
```
Comment (louisebolden) [inline terraform/production/main.tf]: (Same question re. instructional vs informational comments as above :)

**Review by louisebolden — APPROVED** [review]
Looks good.

**Review by humulla — APPROVED** [review]

File: terraform/development/main.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# 7) ENSURE THIS FILE IS PLACED WITHIN A 'terraform' FOLDER LOCATED AT THE ROOT PROJECT DIRECTORY
+
+provider "aws" {
+  region  = "eu-west-2"
+  version = "~> 2.0"
+}
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+locals {
+  application_name = "resident information api" # The name to use for your application
```
Comment (louisebolden) [inline terraform/development/main.tf]: Would we use the kebab-case naming convention (like [here](https://github.com/LBHackney-IT/mosaic-resident-information-api/blob/0d4e5f50bc5c123d6832de5994c653ecaefe4667/terraform/development/main.tf#L40)) for this name also?

Appreciate that's a slightly random example from a VPC definition so may not be relevant here.

Not sure where this local is used, actually - will have a look around the files now!

Comment (elcorbs) [reply terraform/development/main.tf]: We don't seem to be setting it in the other repos so I wasn't sure. Happy to change it as it's probably safer to have it like that. (also never heard the term kebab-case before 😂 )

Comment (makeyw) [reply terraform/development/main.tf]: *goes off and Googles kebab-case* (new one to me, thank you!)

File: terraform/development/main.tf (hunk 1 above)
Comment (elcorbs) [reply terraform/development/main.tf]: Yeah makes sense, the repo does link back to the template which has the instructions in as well.

File: terraform/development/main.tf (hunk 3 above)
Comment (makeyw) [reply terraform/development/main.tf]: Searched for it and found these: camelCase and PascalCase i'd heard of...
But snake_case and kebab-case - not so much!

**Pull request merged**